pkg/handler/eval.go

```diff
@@ -199,6 +200,19 @@ var logEvalResultToDatadog = func(r *models.EvalResult) {
 	)
 }
 
+var logEvalResultToPrometheus = func(r *models.EvalResult) {
+	if config.Global.Prometheus.EvalCounter == nil {
+		return
+	}
+	config.Global.Prometheus.EvalCounter.WithLabelValues(
+		util.SafeStringWithDefault(r.EvalContext.EntityType, "null"),
```

Comment:
not very familiar with the convention of prometheus lable values, does it need to have a name in the value? For example,

```
fmt.Sprintf("flag_id:%s", util.SafeStringWithDefault(r.FlagID, "null"))
```

Otherwise, how can we know if it's variantID or flagID, or something else?

Reply:
@zhouzhuojie that's a bit confusing, yeah - the labels are coded in when the Counter is created here: https://github.com/checkr/flagr/pull/221/files#diff-091ace02b49a46f55f2586cfec89a160R96